lib/project_types/script/layers/infrastructure/languages/project_creator.rb

```diff
@@ -26,8 +26,9 @@ def self.for(
               "wasm" => WasmProjectCreator,
             }
 
-            raise Errors::ProjectCreatorNotFoundError unless project_creators[language]
```

Comment:
I'm guessing we can 🔥  `Errors::ProjectCreatorNotFoundError` and `Errors::TaskRunnerNotFoundError` now?